src/components/CourseForm.jsx

```diff
@@ -1,77 +1,96 @@
-import { React, useState} from 'react';
-import { Button, Modal, Form} from 'react-bootstrap';
+/* eslint-disable valid-jsdoc */
+import {React, useState} from 'react';
+import {Button, Modal, Form} from 'react-bootstrap';
 
 
-async function addCourse(course){
-    const baseUrl = process.env.API_BASE_URL || 'http://localhost:8080';
-    return fetch(`${baseUrl}/api/courses/addCourse`,{
-        headers: {
-            'Content-Type': 'application/json'
-        },
-        method: 'POST',
-        body: JSON.stringify(course)
-    })
-    .then(data=> console.log(data))
-    .catch(err=> console.log(err))
+/**
+ * @params {object} course
+ * @returns fetch
+ */
```

Comment:
this is what `valid-jsdoc` is asking for. But you still ended up ignoring the rule for this file. I suggest ignoring the rule for entire project (See my comment above about exluding it as a rule)